roles/cobbler/templates/triggers/install/post/cephlab_ansible.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+## {{ ansible_managed }}
+set -ex
+name=$2
+export USER=root
+export HOME=/root
+ANSIBLE_CM_PATH=/root/ceph-cm-ansible
+
+# Bail if the ssh port isn't open.
+nc -vz $name 22
```

Comment:
Can we leave some comments about how this is meant to fail silently here the first time it's ran, but then is called again by the rc.local after first reboot?